centaur/src/main/resources/standardTestCases/drs_tests/drs_usa_hca_data_in_jdr.wdl

```diff
@@ -0,0 +1,70 @@
+version 1.0
+
+workflow drs_usa_hca_data_in_jdr {
+    input {
+        File file
+        Int disk_size_required_for_file_gb
+    }
+
+    call localize_drs_with_usa {
+        input:
+            file = file,
+            disk_size_required_for_file_gb = disk_size_required_for_file_gb
+    }
+
+    call skip_localize_drs_with_usa { input: file = file }
+
+    output {
+        String path = localize_drs_with_usa.path
+        String hash = localize_drs_with_usa.hash
+        Float size = localize_drs_with_usa.size
+        String cloud = skip_localize_drs_with_usa.path
+    }
+}
+
+task localize_drs_with_usa {
+    input {
+        File file
+        Int disk_size_required_for_file_gb
+    }
+
+    command <<<
+        echo ~{file} > path
+        md5sum ~{file} | cut -c1-32 > hash
+    >>>
+
+    output {
+        String path = read_string("path")
+        String hash = read_string("hash")
+        Float size = size(file)
+    }
+
+    runtime {
+        docker: "ubuntu"
+        backend: "papi-v2-usa"
+        bootDiskSizeGb: disk_size_required_for_file_gb
```

Comment:
`bootDiskSizeGb` is for handling larger docker images. `disks` is for operating on larger reference and other data files.

Since the HCA file is data I believe you don't need to increase the disk size for downloading and unzipping the docker images.

Reply:
I thought by default there's just a single disk attached.